libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -93,6 +104,18 @@ + (instancetype)sharedManager
         } else {
             sdkManager = [[self alloc] init];
         }
+
+        /*
+         * Checks if an analytics app name has already been set by the app.
+         * If not, fetches the default app name to be used and sets it.
+         */
+        NSString *currentAiltnAppName = [SalesforceSDKManager ailtnAppName];
```

Comment:
@khawkins I didn't do that because if `InstanceClass` is set, it'll call `init` of that class as opposed to this one, in which case the default name won't be set.